modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/spaces/SpaceConnectedSitesModal.tsx

```diff
@@ -3,21 +3,28 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import Autocomplete from '@clayui/autocomplete';
 import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
 import {ClayDropDownWithItems} from '@clayui/drop-down';
+import {ClayInput} from '@clayui/form';
 import ClayModal from '@clayui/modal';
 import ClaySticker from '@clayui/sticker';
 import {openToast} from 'frontend-js-components-web';
+import {ItemSelector} from 'frontend-js-item-selector-web';
 import React, {useEffect, useId, useState} from 'react';
 
 import ConnectedSiteService from '../../common/services/ConnectedSiteService';
 import {Site} from '../../common/types/Site';
 
-const getLocalizedName = (name: Site['name']) => {
+const getLocalizedName = (site: Site) => {
+	if (!site.name_i18n) {
+		return site.name;
+	}
```

Comment:
What brought about the changes in this function... these are fairly large logic chnages.

Reply:
They introduced some changes to make site names localizable. At first by makin the site `name` a map. After a few days `name` was turned back into a string, and `name_i18n` was introduced to hold the translations.

`Liferay.ThemeDisplay.getLanguageId()` and `Liferay.ThemeDisplay.getDefaultLanguageId()` functions retrieve the language as `en_US` while `name_i18n` contains the language codes like `en-US` so thats why I introduced `Liferay.ThemeDisplay.getBCP47LanguageId()`